packages/eslint-plugin/tests/rules/explicit-function-return-type.test.ts

```diff
@@ -1008,24 +1110,27 @@ const func = (value: number) => ({ type: 'X', value } as Action);
       errors: [
         {
           messageId: 'missingReturnType',
-          line: 2,
-          endLine: 2,
+          line: 3,
+          endLine: 3,
           column: 14,
           endColumn: 32,
         },
         {
           messageId: 'missingReturnType',
-          line: 3,
-          endLine: 3,
+          line: 4,
+          endLine: 4,
           column: 14,
           endColumn: 32,
         },
       ],
     },
     {
-      filename: 'test.ts',
       code: `
 const func = (value: number) => ({ type: 'X', value } as const);
+      `,
+      // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
```

Comment:
If I didn't disable this rule and I format the next line correctly (i.e. split it across two lines), then the test fails because my fixer doesn't format the result. Is this what I should be doing? Or is there a better way?